docs.bruno/Test.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Test
+  type: http
+  seq: 1
+}
+
+get {
```

Comment:
Consider adding assertions to your tests to verify the response of the API. This will make your tests more robust and meaningful. [important]